packages/kos-web/demo/src/index.js

```
@@ -0,0 +1,94 @@
+import {
+    Wallet,
+    PathOptions,
+    TransactionChainOptions,
+    toBytes,
+    toString
```

Biome (2.4.6)

[error] Do not shadow the global "toString" property.

(lint/suspicious/noShadowRestrictedNames)